src/remotelibrarymanager.cpp

```diff
@@ -0,0 +1,28 @@
+#include "remotelibrarymanager.h"
+
+RemoteLibraryManager::RemoteLibraryManager()
+: mp_reply(nullptr)
+{
+}
+
+void RemoteLibraryManager::doUpdate(const QUrl &url)
+{
+    QNetworkRequest request(url);
+    if (mp_reply) {
+        mp_reply->abort();
```

Comment:
Should we also call `mp_reply->deleteLater()` ?

Reply:
When mp_reply->abort() is called, it emits a 'finished' signal which is connected to the slot "receivedContent" where I already call mp_reply->deleteLater() so I think not.

https://github.com/kiwix/kiwix-desktop/blob/2bf05ddb6c4a6f0b1fdb2bcd0d2b4d2f0b2630d9/src/remotelibrarymanager.cpp#L8-L28

Reply:
We are good so.